packages/tabs/package.json

```diff
@@ -18,15 +18,16 @@
     "@development-framework/dm-core": ">=1.0.35"
   },
   "files": [
+    "blueprints",
     "dist",
     "src",
```

Comment:
```suggestion
```

Reply:
src is needed to generate documentation: https://equinor.github.io/dm-docs/docs/libraries/@development-framework/dm-core/Components/Dialog. We can't get this from dist alone because comments are removed in the build step (I think, ask Morten for details)